cmd/terramate/cli/cli.go

```diff
@@ -1298,27 +1283,32 @@ func (c *cli) getConfigValue() {
 			fatal(err, "evaluating expression: %s", exprStr)
 		}
 
-		var o []byte
-		if c.parsedArgs.Experimental.GetConfigValue.AsJSON {
-			o, err = json.Marshal(val, val.Type())
-			if err != nil {
-				fatal(err, "converting value %s to json", val.GoString())
-			}
+		c.outputEvalResult(val, c.parsedArgs.Experimental.GetConfigValue.AsJSON)
+	}
+}
+
+func (c *cli) outputEvalResult(val cty.Value, asJSON bool) {
+	var data []byte
```

Comment:
Ah...where is my completely awesome `o` variable ? :-(

Reply:
ah sorry I thought it was mine `o` variable :joy: :joy: